recipes/neuralprophet/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "neuralprophet" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/neuralprophet-{{ version }}.tar.gz
+  sha256: b6f0ecafa8c332c961f071a8664e1f6ba481822012b9cbd9352033cde9b90d4e
+
+build:
+  noarch: python
```

Comment:
This is noarch and you probably don't need that osx-arm64 change even in the feedstock.

Reply:
do I have to get rid othe build no arch as well then?

Reply:
I don't see any compiled extensions in the package, so keep it noarch.